lib/features/family/features/topup/cubit/topup_state.dart

```diff
@@ -0,0 +1,30 @@
+part of 'topup_cubit.dart';
+
+enum TopupStatus { initial, loading, done, error }
+
+class TopupState extends Equatable {
+  const TopupState({
+    this.userGuid = '',
+    this.status = TopupStatus.initial,
```

Comment:
So personally I wouldn't write the topup state this way.

Right now its implied that every state (initial, loading, done, error) has the same fields: userGuid and error.

https://medium.com/@ssindher11/exploring-sealed-classes-in-flutter-241d3e160132

By using a sealed class instead of an enum you can define per state which fields are present, and whether they are required or not. Fields that are shared you define in the sealed class itself and fields that are specific per state you define in the class that's extending the sealed class.